cmd/airflow.go

```diff
@@ -150,6 +157,8 @@ func newAirflowInitCmd() *cobra.Command {
 	}
 	cmd.Flags().StringVarP(&projectName, "name", "n", "", "Name of Astro project")
 	cmd.Flags().StringVarP(&airflowVersion, "airflow-version", "a", "", "Version of Airflow you want to create an Astro project with. If not specified, latest is assumed. You can change this version in your Dockerfile at any time.")
+	cmd.Flags().StringVarP(&fromTemplate, "from-template", "t", "", "Template name that you want to use to create the local astro project. Possible values can be etl, dbt-on-astro, generative-ai and learning-airflow. Please note template based astro project use latest runtime version and airflow-version flag will be ignored when creating a project with template flag")
```

Comment:
maybe either we don't specify the option in the description if that is subject to change, or we call the `airflow.FetchTemplateList` as part of the `newAirflowInitCmd` itself so that the description could be dynamic based on response, wdyt?

Reply:
Thanks @neel-astro ! I've updated the description of flag to remove the template names